Update ingress apiVersion (#11)
ambimax/action-docker-deploy: README.md

          values_file: ./values.yml
          undeploy: "true"
```

### [kubernetes-namespace](test/09-kubernetes-namespace)

> Deploy a container to a digitalocean kubernetes cluster under a different namespace

Reviewer: ```suggestion
Deploy a container on a digitalocean kubernetes cluster within a specific namespace
```

```yml
      - uses: ambimax/action-docker-deploy@v2
        with:
          image: containous/whoami
          host: ${{ secrets.EXAMPLE_HOST }}
          digitalocean_token: ${{ secrets.EXAMPLE_DIGITALOCEAN_TOKEN }}
          digitalocean_cluster: ${{ secrets.EXAMPLE_DIGITALOCEAN_CLUSTER }}
          enable_commit_comment: true
          github_token: ${{ secrets.GITHUB_TOKEN }}
          namespace: kube-system
```

### [kubernetes-namespace-undeploy](test/09-kubernetes-namespace-undeploy)

> Deploy a container to a digitalocean kubernetes cluster under a different namespace

Reviewer: ```suggestion
Undeploy a container on a digitalocean kubernetes cluster within a specific namespace
```
